<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_alarm_dist</class>
 <widget class="QDialog" name="Dialog_alarm_dist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>383</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>331</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>321</width>
     <height>331</height>
    </size>
   </property>
   <property name="title">
    <string>Modo distancia:</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="suffix">
     <string> s</string>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>178</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="suffix">
     <string> s</string>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>232</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Límite superior:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar durante:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>78</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GPIO 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPIO 27</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar Alarma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>282</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Límite inferior:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar alerta en:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>132</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar salida pasados:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_alarm_dist</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_alarm_dist</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
